flutter_app/lib/deviceVars.dart

```diff
@@ -0,0 +1,10 @@
+class GlobalData {
+  //string variables for other screens
+  static String app1Name = "";
```

Comment:
I'm not the biggest fan of these varaible names. But maybe it would be better to have an array of strings so that you don't assume that only three apps will be displayed.

Reply:
Maybe static String[] appNames = []

Reply:
the only use of this class is to pass data to the main page. it would be easy to go back and change this in the future but for now displaying the top 3 apps is the only use hence three variables.

Reply:
keeping them in.